kafka-cron/grafana_datasources.yml

```diff
@@ -0,0 +1,9 @@
+## grafana_datasources.yml ##
```

Comment:
you should be able to export your dashboards as json for review https://grafana.com/docs/grafana/v9.0/dashboards/export-import/